DOC: add unqiue value counts example to groupyby guide (#37847)
pandas-dev/pandas: doc/source/user_guide/groupby.rst

   grouped.describe()

> Another aggregation example is to compute the number of unique values of a group. This is similar to the ``value_counts`` function, execpt that ``value_counts`` return the number of all values instead of the number of unique values.

Reviewer: ```suggestion
Another aggregation example is to compute the number of unique values of each group. This is similar to the ``value_counts`` function, except that it only counts unique values.
```

.. ipython:: python

   ll = [['foo', 1], ['foo', 2], ['foo', 2], ['bar', 1], ['bar', 1]]

>    pd.DataFrame(ll, columns=["A", "B"]).groupby("A")["B"].nunique()

Reviewer 2: can you first show the input df, then do the groupby.nunique on the next line

>    pd.DataFrame(ll, columns=['A', 'B']).groupby('A').nunique()

Reviewer: ```suggestion
   pd.DataFrame(ll, columns=["A", "B"]).groupby("A")["B"].nunique()
```
